### ISSUE TITLE
Initialization/load flow of viewer is unstable

### ISSUE THREAD

**Issue by sanmont3drepo** [3drepo/3drepo.io#4576]
### Description
The initialization of the viewer is a little shaky in some cases it doesnt initialize correctly. 

- This is happening in: both
- This requires a hotfix to production: no 

### Steps to replicate
- Enter a container of a teamspace
- click on a notification for a container of other teamspace
 
-## Current Behaviour
 - Loading stops at 75% 
 - pins are not showing

### Expected Behaviour
- Loading should be completed
- Pins should be showing

### Pictures/ Gifs of the error
<img width="1218" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/a7cab6fe-f002-4234-8ae1-a99cb347d527">

**Comment by AsiteZane** [comment]
@carmenfan - this is complete now based on #4499 ? I'm assuming I can close?

**Comment by carmenfan** [comment]
> @carmenfan - this is complete now based on #4499 ? I'm assuming I can close?

No, this is spun out of 4405 as sometihng found that was out of scope. This only happens on the situation santiago described, which is to do with the notification tab that you want to revamp, so it might just be something that should be tagged against that so we check and fix when we implement the new one.

**Comment by AsiteZane** [comment]
@carmenfan - thank you, i'll tag it into the epic
